FileAPI/url/url-format.any.js

```diff
@@ -0,0 +1,63 @@
+const blob = new Blob(['test']);
+const file = new File(['test'], 'name');
+
+test(() => {
+  const url_count = 10000;
+  let list = [];
+
+  for (let i = 0; i < url_count; ++i)
+    list.push(URL.createObjectURL(blob));
+
+  list.sort();
+
+  for (let i = 1; i < list.length; ++i)
+    assert_not_equals(list[i], list[i-1], 'generated Blob URLs should be unique');
+}, 'Check whether generated Blob URLs are unique');
```

Comment:
nit: In the end, good tests with any naming convention are better than no tests (hence why this is a nit), but I hope you'll at least consider removing the "{Check, Verify} {if, whether} " prefixes. I think that straight out deleting those 2 words in every test description where they're present will still result in a readable description that's a bit smaller.

Reply:
Done, thanks for the review!